server/command.go

```diff
@@ -44,14 +44,13 @@ func getCommand() *model.Command {
 	}
 }
 
-func (p *Plugin) getCommandResponse(responseType, text string) *model.CommandResponse {
-	return &model.CommandResponse{
-		ResponseType: responseType,
-		Text:         text,
-		Username:     GITHUB_USERNAME,
-		IconURL:      p.getConfiguration().ProfileImageURL,
-		Type:         model.POST_DEFAULT,
+func (p *Plugin) postCommandResponse(args *model.CommandArgs, text string) {
+	post := &model.Post{
+		UserId:    p.BotUserID,
+		ChannelId: args.ChannelId,
+		Message:   text,
 	}
+	p.API.SendEphemeralPost(args.UserId, post)
 }
 
 func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
```

Comment:
I would like (2/5) to refactor this into something like:
```suggestion
func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
	p.executeCommand(c *plugin.Context, args *model.CommandArgs)
	return &model.CommandResponse{}, nil
```

and remove the return value from `executeCommand`.

But I'm fine with opening a ticket for this instead of doing this in this PR.

Reply:
Separate ticket.